assets/icons/preview.html

```diff
@@ -0,0 +1,687 @@
+<html>
```

Comment:
I see. Thanks for the clarification.

All the files under the `assets` folder are available to clients in the download package. It doesn't sound like the clients will need this preview file.

I suggest we either a) git ignore this file b) keep this file but add a comment at the beginning so internal developers and clients know the purpose of it.

Reply:
I will add this file to .gitignore.